src/IB/IBFEMethod.cpp

```diff
@@ -787,6 +852,10 @@ IBFEMethod::interpolateVelocity(const int u_data_idx,
                                                         /*close_F*/ false,
                                                         /*close_X*/ false);
     }
+
+    // note that FEDataManager only reads (no writes) Eulerian info so no data
```

Comment:
Do we spread to the scratch hierarchy?

Reply:
I should clarify this comment: I meant `FEDataManager only reads (no writes) Eulerian data during interpolation`.